src/services/infra/DynamoDBClient.ts

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable consistent-return */ // todo remove this
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb"
+import {
+  DynamoDBDocumentClient,
+  PutCommand,
+  GetCommand,
+  UpdateCommand,
+  DeleteCommand,
+  UpdateCommandInput,
+} from "@aws-sdk/lib-dynamodb"
+import autoBind from "auto-bind"
+
+import { MessageBody } from "@root/../microservices/site-launch/shared/types"
+import logger from "@root/logger/logger"
+
+export interface UpdateParams {
+  TableName: string
+  Key: Record<string, string>
+  UpdateExpression: string
+  ExpressionAttributeNames: Record<string, string>
+  ExpressionAttributeValues: Record<string, string>
+}
+export default class DynamoDBDocClient {
+  private readonly dynamoDBDocClient: DynamoDBDocumentClient
+
+  constructor() {
+    const marshallOptions = {
+      // Whether to automatically convert empty strings, blobs, and sets to `null`.
+      convertEmptyValues: false,
+      // Whether to remove undefined values while marshalling.
+      removeUndefinedValues: true,
+      // Whether to convert typeof object to map attribute.
+      convertClassInstanceToMap: false,
+    }
+
+    const unmarshallOptions = {
+      // Whether to return numbers as a string instead of converting them to native JavaScript numbers.
+      wrapNumbers: false,
+    }
+
+    this.dynamoDBDocClient = DynamoDBDocumentClient.from(
+      new DynamoDBClient({ region: "ap-southeast-1" }),
```

Comment:
It's better to use env variable for `region` than hardcoding it in the code here.

Reply:
4d2bcc09